packages/react-router/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 2.0.2
+
+### Patch Changes
+
+- Updated dependencies []:
```

Comment:
Do you know why there isn't a commit hash here?

Reply:
Huh, it seems to affect `react-router` and `@shopify/react-cookie`, both are 3 layers of transitivity from the root change (these two -> react-network -> network). Perhaps it the commit data gets lost when traversing more than one level deep?

Reply:
🤷 🚢